### ISSUE TITLE
Ubuntu 14.04 support

### ISSUE THREAD

**Issue by justindossey** [Alfresco/chef-alfresco#117]
Though the README says Ubuntu 14.04 is supported, I have found a number of changes are necessary to make this work (and I still haven't succeeded).
- additional tomcat packages have different names on Ubuntu.  In my `.kitchen.local.yml` I need:

```
    attributes:
      tomcat:
        additional_tomcat_packages:
          - libapr1
          - libtcnative-1
```

It is also necessary to set `node['apt']['compile_time_update'] = true` so that `apt-get update` runs prior to package installation.

Also, the tomcat template appears to have an error, leading to this in the converge:

```
        * directory[/var/lib/tomcat7-alfresco/webapps] action create[2016-09-16T18:28:05+00:00] INFO: Processing directory[/var/lib/tomcat7-alfresco/webapps] action create (/tmp/kitchen/cache/cookbooks/tomcat/providers/instance.rb line 55)
```

followed shortly by

```
         * link[/var/lib/tomcat7-alfresco/webapps] action create[2016-09-16T18:28:05+00:00] INFO: Processing link[/var/lib/tomcat7-alfresco/webapps] action create (/tmp/kitchen/cache/cookbooks/tomcat/providers/instance.rb line 86)
       [2016-09-16T18:28:05+00:00] DEBUG: Providers for generic link resource enabled on node include: [Chef::Provider::Link]
       [2016-09-16T18:28:05+00:00] DEBUG: Provider for action create on resource link[/var/lib/tomcat7-alfresco/webapps] is Chef::Provider::Link


           ===============================================================================
           Error executing action `create` on resource 'link[/var/lib/tomcat7-alfresco/webapps]'
           ================================================================================

           Errno::EISDIR
           -------------
           Is a directory @ unlink_internal - /var/lib/tomcat7-alfresco/webapps

           Resource Declaration:
           ---------------------
           # In /tmp/kitchen/cache/cookbooks/tomcat/providers/instance.rb

            86:       link "#{new_resource.base}/#{name}" do
            87:         to new_resource.instance_variable_get("@#{attr}")
            88:       end
            89:     end
```

**Comment by enzor** [comment]
Hi Justin,

thanks for reporting this issue, will try a local run and see what are the problems.
Can I look at your copy of .kitchen.local.yml, if possible?

**Comment by justindossey** [comment]
Sure, here's the file.  You'll need to set repository auth info to make it work for you.
```
---
# Update the username and password (below) and rename to .kitchen.local.yml.
# Add the IP address of your instance to /etc/hosts as alfresco.local or alfresco-ubuntu.local for
# convenience.
provisioner:
  name: chef_zero
  log_level: debug
platforms:
  - name: centos-7.2
    driver_config:
      network:
      - ["private_network", {ip: "192.168.33.23"}]
  - name: ubuntu-14.04
    driver_config:
      network:
      - ["private_network", {ip: "192.168.33.33"}]

suites:
  - name: me
    includes: ubuntu-14.04
    run_list:
      - recipe[alfresco::default]
      - recipe[alfresco::redeploy]
    data_bags_path: test/integration/data_bags
    attributes:
      apt:
        compile_time_update: true
      tomcat:
        additional_tomcat_packages:
          - libapr1
          - libtcnative-1
      name: chef-alfresco-me
      hostname: alfresco-me-test
      artifact-deployer:
        maven:
          timeout: 3600
          repositories:
            internal:
              url: https://artifacts.alfresco.com/nexus/content/groups/private
              username: X
              password: X
      nginx:
        public_hostname: alfresco-ubuntu.local
      alfresco:
        use_nossl_config: true
        public_hostname: alfresco-ubuntu.local
        # valid versions: 5.1.1.4, 5.0.2.5
        version: "5.1.1.4"
        components:
          - tomcat
          - repo
          - share
          - mysql
          - solr
          - haproxy
          - nginx
        edition: enterprise
        ssl_enabled: false
  - name: me-centos
    includes: centos-7.2
    run_list:
      - recipe[alfresco::default]
      - recipe[alfresco::redeploy]
    data_bags_path: test/integration/data_bags
    attributes:
      name: chef-alfresco-me
      hostname: alfresco-me-test
      artifact-deployer:
        maven:
          timeout: 3600
          repositories:
            internal:
              url: https://artifacts.alfresco.com/nexus/content/groups/private
              username: X
              password: X
      nginx:
        public_hostname: alfresco.local
        use_nossl_config: true
      alfresco:
        public_hostname: alfresco.local
        # valid versions: 5.1.1.4, 5.0.2.5
        version: "5.1.1.4"
        certs:
          ssl_databag_item: local
        components:
          - tomcat
          - repo
          - share
          - mysql
          - solr
          - haproxy
          - nginx
        edition: enterprise
        ssl_enabled: false
```

**Comment by enzor** [comment]
Hi Justin,

I am sorry for the delay of this reply, but work has been crazy lately.
I am doing a whole refactor of the project, and your issue will be addressed properly in a second moment, once the v0.8 milestone has been reached.

Will updated you as soon as we can, we have all the interest to make this project compatible on Ubuntu
